### ISSUE TITLE
Add slack badge to ivado-internal

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#729]
Like they do in torchIO: 
![image](https://user-images.githubusercontent.com/2482071/111684347-ff51ee80-87fc-11eb-8a53-d5b091dd5dcd.png)

Cool idea to attract more ppl and brainstorming.

**Comment by copperwiring** [comment]
@jcohenadad This would nice to get more people to brainstorm but then this slack should also be open to public? Do we have channels open to public? Is this a feature of future?

**Comment by jcohenadad** [comment]
> @jcohenadad This would nice to get more people to brainstorm but then this slack should also be open to public? Do we have channels open to public? Is this a feature of future?

Yes, we can have channels open to public but you made a very good point. At the end it might not be super useful to have a slack badge (at least for now).
